extension/core_functions/include/core_functions/aggregate/distributive_functions.hpp

```diff
@@ -299,4 +299,14 @@ struct SumNoOverflowFun {
 	static AggregateFunctionSet GetFunctions();
 };
 
+struct MaxIntersectionsFun {
```

Comment:
This is a holistic aggregate (it requires all the data) so it should be in the holistic section.